_pages/webextensions.md

```diff
@@ -12,160 +12,173 @@ parent: activities
 
-__Part I__
+### Third Meeting: APIs in Action, Part 2
+During the third event, participants will explore how their extensions can use the [tabs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs), [sidebarAction](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/sidebarAction), and [webRequest](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/webRequest) APIs to interact with the browser. You can make a copy of the presentation [APIs in Action, Part Two](https://docs.google.com/presentation/d/1_iP7TvHQxKbMynh3tjI1kHGh0gZ5ZElG-K9RYvYjbMk/edit), to customize and share with your group. 
 
-* Give a quick introduction to attendees about what we are doing and why it is important.
-  * “Welcome to the \[name of event\]! I am \[name\]. Today, we are going to customize our Firefox browsers by developing our own extensionsto solve real problems in our own lives. Add-ons are a great way to add extra features to your browser, like blocking time-wasting websites during study hours, or turning images into cat pictures. With WebExtensions APIs, it’s now easier to create add-ons that are compatible with Firefox, Chrome, and Opera. Add-ons help keep users to be in control of their online experience, so your support in building them and the community around them matter.”
-* Ask attendees to tweet about their attendance. Here is a suggested message:
-  * I’m creating my own browser extension using #WebExtensions! #MozActivate
-* Ask participants if they already use add-ons. If they do, ask them to share what add-ons they use and how they are used.
+We recommend the following agenda for this meeting: 
 
-__Part II__
+* 1st half of the meeting: provide an overview and demonstration of the following topics:
 
-1. Now it’s your turn. Using the tutorial on MDN [build your first extension](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Your_first_WebExtension).
-2. After creating your first extension, move on to Step 5 or follow the tutorial to [build a more complex extension](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Your_second_WebExtension).
-3. Once everyone in the group has made an extension, take a few minutes to debrief. While you debrief, you can learn more about the [anatomy of an extension](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Anatomy_of_a_WebExtension), explore [more examples of APIs in action](https://github.com/mdn/webextensions-examples), continue to Part III, or conclude the event.
+| Topic			| Extension Examples			| Videos 			|
+| ---------------------	|:-------------------------------------:|:-----------------------------:|
+| [Tabs API](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs)		| [Tabs Tabs Tabs](https://github.com/mdn/webextensions-examples/tree/master/tabs-tabs-tabs)			| [Getting Started with Tabs](https://www.youtube.com/watch?v=GS1nM55iKZo&t=612s&list=PLdIyyZgtQZ7CJR5XFB5xvZH0oaBIT_gTD&index=5)	|
```

Comment:
can you fix this table as well? The first one looks good now, so the format should be the same as the first one.

Reply:
(using spaces instead of tabs in the title row fixes the issue)

Reply:
When checking out your branch, I still see it wrong:

![screen shot 2018-04-03 at 18 51 30](https://user-images.githubusercontent.com/330324/38263306-108e098a-3770-11e8-8069-0a4294642023.png)

I'm solely use the command listed in the README: ```bundle exec jekyll serve --config ./_config-dev.yml```

When I look at the code, I see that there are tabs:

![screen shot 2018-04-03 at 18 54 46](https://user-images.githubusercontent.com/330324/38263479-86749c68-3770-11e8-9aad-dc34d71b54c4.png)

As you can see on the second line (the separators) there is a tab before the second | on the row. Removing that fixes the issue. Here's the diff for it:

```
-| ---------------------        |:-------------------------------------:|:-----------------------------:|
+| --------------------- |:-------------------------------------:|:-----------------------------:|
```
Hope that helps :)

Reply:
In any case, it would be great to be only using spaces instead of tabs, no matter if it works with or without or not :)